composer.json

```diff
@@ -22,6 +22,7 @@
     "greenpeace/planet4-plugin-medialibrary" : "dev-master",
     "wikimedia/composer-merge-plugin": "1.4.1",
     "wpackagist-plugin/akismet": "4.*",
+    "wpackagist-plugin/cloudflare" : "3.8.5",
```

Comment:
Do you think we should go for `3.8.*`? From their [Changelog](https://github.com/cloudflare/Cloudflare-WordPress/releases) it seems relatively safe.

Reply:
Idk, it feels safer to review any changes they make before we upgrade. Some parts of the plugin have very poor architecture and so seem quite prone to bugs, even with small changes.